agent/common.sh

```diff
@@ -11,8 +11,14 @@ export CLUSTER_HOST_PREFIX_V4="24"
 export EXTERNAL_SUBNET_V6="fd2e:6f44:5dd8:c956::/64"
 
 if [ -n "$MIRROR_IMAGES" ]; then
-    # We're going to be using a locally modified release image
-    export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="${LOCAL_REGISTRY_DNS_NAME}:${LOCAL_REGISTRY_PORT}/localimages/local-release-image:${OPENSHIFT_RELEASE_TAG}"
+
+    export MIRROR_IMAGE_URL_SUFFIX="localimages/local-release-image"
+
+    if [ "${OC_MIRROR}" == true ]; then
+        export MIRROR_IMAGE_URL_SUFFIX="openshift/release-images"
```

Comment:
any specific reason to change the target repo? Keeping the default one will make  easier to use this code in future also for the standard flow

Reply:
> any specific reason to change the target repo? Keeping the default one will make easier to use this code in future also for the standard flow

Running the `oc mirror` command as in oc_mirror.sh will result in an ICSP using `openshift/release-images` which is why the target has to be set like that.
  repositoryDigestMirrors:
  - mirrors:
    - virthost.ostest.test.metalkube.org:8443/openshift/release
    source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
  - mirrors:
    - virthost.ostest.test.metalkube.org:8443/openshift/release-images
    source: quay.io/openshift-release-dev/ocp-release